rootfs/api/models/app.py

```diff
@@ -33,6 +34,26 @@
 logger = logging.getLogger(__name__)
 
 
+def is_experimental_native_ingress():
+    """
+    Will determine the value of the experimental_native_ingress flag
+    """
+    is_ingress = os.getenv('EXPERIMENTAL_NATIVE_INGRESS', False)
+    if not is_ingress:
+        return False
+    return True
+
+
+def get_experimental_native_ingress_hostname():
+    """
+    Will return the experimental native ingress platform_domain value, or an empty string
+    """
+    host = os.getenv('EXPERIMENTAL_NATIVE_INGRESS_HOSTNAME', "")
```

Comment:
It would be a little simpler to test for truthy/falsey rather than an empty string explicitly IMHO:
```python
host = os.getenv('EXPERIMENTAL_NATIVE_INGRESS_HOSTNAME')
if not host:
    raise ServiceUnavailable("Unable to find hostname")
return host
```